### DISCUSSION TITLE
Cupones T3

### DISCUSSION THREAD

**Discussion by AleSierra26** [IIC2133-PUC/2024-1#185] [Tarea 3] [answered]
Holaaa, queria saber si van a habilitar los cupones de dias extra para la tarea? (puede ser que ya lo hayan hecho y no lo haya visto 😢)

**Comment by MaxAl100** [comment]
A mi conocimiento no los han habilitado, ya que (al menos en las últimas tareas) crean un anuncio en Canvas con el link para postular a los cupones extra. Este anuncio, que igual se envía por Telegram, no lo han enviado todavía.
Pero: Según el enunciado sí van a habilitar cupones para días extra, por lo que hay que prestar atención dentro de los próximos días cuando publiquen la información

**Reply by AleSierra26** [reply]
Vale, gracias maestro🙏

**Comment by benjavicente** [comment, marked as answer]
Hola! Aquí está el link del formulario que mandamos por avisos ayer, por si acaso:

https://forms.gle/gQFGCpEVUFnXKiQm8
